lunar/distribution.yaml

```diff
@@ -675,5 +675,21 @@ repositories:
       url: https://github.com/ros/std_msgs.git
       version: groovy-devel
     status: maintained
+  urdfdom_py:
+    doc:
+      type: git
+      url: https://github.com/ros/urdf_parser_py.git
+      version: 0.3.2
```

Comment:
I was just copying what was in kinetic here (mostly because I wasn't entirely sure what this doc section really meant).  Does bloom not update the doc version when doing a release?